### ISSUE TITLE
Sticky Footer/Header

### ISSUE THREAD

**Issue by stefanreimers** [emilsjolander/StickyScrollViewItems#20]
I am working on a vertical ScrollView with the following contents: an image with some text at top, followed by a title TextView and lots of text at the bottom. Like this:

---

image
text
## title textview

lots of text
...
## ...

I want the title in the middle to be sticky and your library works like a charm. But it would be perfect if the sticky TextView would start at bottom of the screen (always visible), even if the text/image above are taking more space than fits on the screen. On scrolling down the title should go up and stay sticky at the top - as is does right now.

So basically I want the sticky TextView to be visible at all times just moving up and down during scrolling. This might be a special case because the layout is simple with only one sticky view and two none sticky ones.

Is that possible?
